custom_dc/build_image.md

```diff
@@ -141,14 +141,14 @@ To upload and deploy the container to the Cloud, see [Deploy a custom instance t
 
 ## Run the container with the local SQLite database
 
-To start the services using the locally built repo. If you have made changes to any of the UI components, be sure to map the `custom` directories to the Docker `workspace` directory.
+Start the services using the locally built repo. If you have made changes to any of the UI components (or directories), be sure to map the `custom` directories (or alternative directories) to the Docker `workspace` directory.
 
 <pre>  
 docker run -it \
---env-file $PWD/custom_dc/sqlite_env.list \
+--env-file $PWD/custom_dc/env.list \
 -p 8080:8080 \
 -e DEBUG=true \
-[-v $PWD/custom_dc/<var>CUSTOM_DATA_DIRECTORY</var>:/userdata \] 
+-v <var>OUTPUT_DIRECTORY</var>:/<var>OUTPUT_DIRECTORY</var> \
```

Comment:
Remove the slash `<var>OUTPUT_DIRECTORY</var>` on the RHS of the colon. Since it's a placeholder, let's just make sure that the paths on both sides of the colon are the same.

Reply:
Done.